lntest/itest/assertions.go

```diff
@@ -1789,3 +1789,66 @@ func assertChannelPolicyUpdate(t *testing.T, node *lntest.HarnessNode,
 		), "error while waiting for channel update",
 	)
 }
+
+func transactionInWallet(node *lntest.HarnessNode, txid chainhash.Hash) bool {
+	txStr := txid.String()
+
+	txResp, err := node.GetTransactions(
+		context.Background(), &lnrpc.GetTransactionsRequest{},
+	)
+	if err != nil {
+		return false
+	}
+
+	for _, txn := range txResp.Transactions {
+		if txn.TxHash == txStr {
+			return true
+		}
+	}
+
+	return false
+}
+
+func assertTransactionInWallet(t *testing.T, node *lntest.HarnessNode, txID chainhash.Hash) {
```

Comment:
nit: line too long

Reply:
nit: this kind of comment can easily be automated, saving a lot of time for things only humans can do. I don't know a lot about go, but I'm pretty sure you can fail your build if there's a long line (assuming you have strict syntax rules that you want to obey).

Reply:
good point. There should exist such an action, will check.